tensorflow-core/tensorflow-core-platform/pom.xml

```diff
@@ -41,25 +41,25 @@
       <groupId>org.tensorflow</groupId>
       <artifactId>tensorflow-core-native</artifactId>
       <version>${project.version}</version>
-      <classifier>${javacpp.platform.linux-x86_64.extension}</classifier>
+      <classifier>linux-x86_64</classifier>
```

Comment:
Maybe we should keep using properties but the with an extension here (e.g. `${javacpp.platform.linux-x86_64`), just so we know the values are consistent. Not strong opinion either way.